apps/explorer/priv/repo/migrations/20211115164817_add_check_for_bytecode_actuality.exs

```diff
@@ -0,0 +1,13 @@
+defmodule Explorer.Repo.Migrations.AddCheckForBytecodeActuality do
+  use Ecto.Migration
+
+  def change do
+    alter table(:smart_contracts) do
+      add(:is_changed_bytecode, :boolean, default: false)
+      # subtracting 1 day to perform first check
+      add(:bytecode_checked_at, :"timestamp without time zone",
```

Comment:
`"timestamp without time zone"` is a valid pg column type??

Reply:
Yeah, I'm not sure why they decided to use it in this case though.